saltstack/base/php/files/etc/php/7.1/mods-available/xdebug.ini

```diff
@@ -1,7 +1 @@
-zend_extension=xdebug.so
-xdebug.remote_enable=0
-xdebug.remote_host=10.10.0.1
```

Comment:
This line needs to stay but must be changed to `xdebug.client_host=10.10.0.1`

Reply:
ok